apps/projects/management/commands/reset_insights_table.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from argparse import ArgumentParser
+
+from django.core.management.base import BaseCommand
+
+from apps.projects.insights import create_insights
+from apps.projects.models import Project
+
+logger = logging.getLogger(__name__)
```

Comment:
@hklarner i added logger for all the apps in apps/__init__ so we can import it from there so no need to declare it in every file we call it.